mods/ascent/mobs/nymph/nymph_holder.dm

```diff
@@ -0,0 +1,17 @@
+/obj/item/holder/ascent_nymph
+	origin_tech = "{'magnets':3,'biotech':5}"
+	slot_flags = SLOT_HEAD | SLOT_OCLOTHING | SLOT_HOLSTER
+	armor = list(
+		bio = ARMOR_BIO_RESISTANT, 
+		rad = ARMOR_RAD_SHIELDED
+	)
+
+// Yes, you can wear a nymph on your head instead of a radiation mask.
```

Comment:
Why do Ascent get radiation blocking behavior? It's from diona nymphs, wondering if you just copied it or if it was deliberate.

Reply:
I thought ascent were also radiation resistant because of their shells? If not I'll remove.